src/tests/unit/tests/DetailsView/components/__snapshots__/adhoc-issues-test-view.test.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`AdhocIssuesTestView should return issues table with scanning to false 1`] = `"<IssuesTable deps={{...}} title=\\"test title\\" issuesTableHandler={{...}} issuesEnabled={true} issueTrackerPath={[undefined]} violations={{...}} issuesSelection={{...}} selectedIdToRuleResultMap={{...}} pageTitle={[undefined]} pageUrl={[undefined]} scanning={false} toggleClickHandler={[Function: clickHandlerStub]} visualizationConfigurationFactory={[undefined]} featureFlags={[undefined]} scanResult={{...}} reportGenerator={{...}} userConfigurationStoreData={[undefined]} />"`;
+exports[`AdhocIssuesTestView should return issues table with scanning to false 1`] = `"<IssuesTable deps={{...}} title=\\"test title\\" subtitle={[undefined]} issuesTableHandler={{...}} issuesEnabled={true} issueTrackerPath={[undefined]} violations={{...}} issuesSelection={{...}} selectedIdToRuleResultMap={{...}} pageTitle={[undefined]} pageUrl={[undefined]} scanning={false} toggleClickHandler={[Function: clickHandlerStub]} visualizationConfigurationFactory={[undefined]} featureFlags={[undefined]} scanResult={{...}} reportGenerator={{...}} userConfigurationStoreData={[undefined]} />"`;
```

Comment:
should we modify the test for adhoc-issues-test-view to make the subtitle change more clear in snapshots?